### PR TITLE
Implement a draft version of the IdentityKit [sc-9867]

### PR DESCRIPTION
**Pull request by dmitriiIdentityLabs**

### CODE REVIEW

File: docs/pages/docs/packages/identity-kit/icrc-31-get-principals.mdx [hunk 1]
```diff
@@ -24,10 +24,12 @@ applications to retrieve information about the identities managed by a signer, s
 authentication process. This standard is pivotal in ensuring that users can manage their digital
 identities efficiently and securely.
 
-<Tabs items={['Method Call', 'Response Type']} defaultIndex="0">
+<Tabs items={['Method Call', 'Request Type', 'Response Type']} defaultIndex="0">
 <Tabs.Tab>
 ```typescript filename="Method Call"
-const response: Response = await connection.getPrincipals()
+const response: Response = await connection.getPrincipals({
+  version: "1"
```
Comment (plitzenberger) [inline docs/pages/docs/packages/identity-kit/icrc-31-get-principals.mdx:31]: Should that be the default?

Comment (dmitriiIdentityLabs) [reply docs/pages/docs/packages/identity-kit/icrc-31-get-principals.mdx:31]: That's my question to you. It's not quite clear for me the process handling of version, I see it in the docs and bring into our docs. Shall I remove the version field for all our endpoints?
image

Comment (frederikrothenberger) [reply docs/pages/docs/packages/identity-kit/icrc-31-get-principals.mdx:31]: Just to chime in here: Version handling can absolutely be discussed on a standards level. I.e. the ICRC-1, 2 and 3 do not have such a version field.

Maybe we the whole versioning mechanism could be dropped and replaced by the `supported_standards` thing? I.e. if you need a version 2 of something, put it in a separate standard and introduce a _new_ method?

Comment (dmitriiIdentityLabs) [reply docs/pages/docs/packages/identity-kit/icrc-31-get-principals.mdx]: Aw, good catch! I was trying to bring the docs in a consistent way and thought it had been discussed and decided upon. As we agreed with Philipp recently, we could set the version field on the config method globally. Additionally, we might provide a default value to make it easier for users. I have made the necessary changes accordingly

**Review by plitzenberger — DISMISSED** [review]
lgtm

**Pull request merged**